lib/active_record/connection_adapters/oracle_enhanced/schema_statements.rb

```diff
@@ -504,6 +505,22 @@ def default_trigger_name(table_name)
           "#{table_name.to_s[0,table_name_length-4]}_pkt"
         end
 
+        def rebuild_primary_key_index_to_default_tablespace(table_name, options)
+          tablespace = default_tablespace_for(:index)
+
+          return unless tablespace
+
+          index_name = Base.connection.select_value(
+            "SELECT index_name FROM all_constraints
+                WHERE table_name = #{quote(table_name.upcase)}
+                AND constraint_type = 'P'
+                AND owner = SYS_CONTEXT('userenv', 'current_schema')")
+
+          return unless index_name # schema_migrations.id does not have index name
```

Comment:
Be actually, a table does not have the primary key in an application then error occurred.